docker/Dockerfile

```diff
@@ -10,18 +10,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+FROM anapsix/alpine-java:8_jdk AS build
+WORKDIR /home/gradle/project
+ADD . .
+RUN ./gradlew jar
 
 FROM anapsix/alpine-java
 
 MAINTAINER coffeepac@gmail.com
 
 WORKDIR /opt/kafka-monitor
-ADD build/ build/
+COPY --from=build /home/gradle/project/build build/
```

Comment:
Should't this use relative paths rather than absolute paths?

Reply:
The line above is set to the working folder `WORKDIR`.
Relative paths are used below this change.
Change everything or leave the consistency?